### ISSUE TITLE
AqaraHub cant run from other folder

### ISSUE THREAD

**Issue by tribukhouski** [Frans-Willem/AqaraHub#45]
when I copy AqaraHub to other folder it can not run
`root@mdmserver:/tmp# ./AqaraHub -p /dev/ttyACM0 `
<INFO> [Main] Serial port: /dev/ttyACM0
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::property_tree::info_parser::info_parser_error> >'
  what():  ../clusters.info: cannot open file for reading

**Comment by Frans-Willem** [comment]
AqaraHub needs the clusters.info file present somewhere. Adding the --cluster-info path/to/clusters.info command should fix things.

Let me know if that solved your issue :)

**Comment by tribukhouski** [comment]
yes, thanks, bro. I wanted to quickly quickly write a service for systemd and did not look for  the parameters. now everything works fine
